<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UF_LoginAsAdmin</class>
 <widget class="QWidget" name="UF_LoginAsAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BACKGROUND-COLOR: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="PB_UserManager">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change your password, create new users, and delete existing users&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">BACKGROUND-COLOR: rgb(160, 238, 252);
BORDER-COLOR: rgb(160, 238. 252);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
   </property>
   <property name="text">
    <string>Return to General Login</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TB_Password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Confirm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm username and password to log in&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">BACKGROUND-COLOR: rgb(170, 255, 127);
BORDER-COLOR: rgb(170, 255, 127);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="LAB_Username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="LAB_Password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TB_Username">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
   </property>
  </widget>
  <widget class="QLabel" name="LAB_Login">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">BACKGROUND-COLOR: rgb(208, 248, 255);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Login as Admin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
